services/processor/native/call.go

```diff
@@ -136,7 +136,14 @@ func (s *service) prepareMethodInputArgsForCall(methodInstance types.MethodInsta
 	}
 
 	if i < methodType.NumIn() {
+		if methodType.NumIn()-1 == i { // missing a single argument
```

Comment:
for non-variadic arrays (`[]byte`) in the last element you will allow the function call to proceed if the array was not included.

Reply:
will `[][]byte` work in the last argument?

Reply:
Resolved.
[][]byte will not work because after introduction of `isVariadic` call you won't be able to pass `([]byte, []byte)` as `[][]byte` (not sure if you could before).